acceptance/Gemfile.lock

```diff
@@ -0,0 +1,112 @@
+GIT
```

Comment:
That's a good question since we trying to test "latest" in regards to version pinning.  I recall at one time we decided to keep the lock file, but for the focus on acceptance maybe we shouldn't.